src/utils/enums/webhook-events.ts

```diff
@@ -0,0 +1,6 @@
+export enum WebhookEvents {
+    Completed = 'invoice.completed',
+    Cancelled = 'invoice.cancelled',
+    Expired = 'invoice.expired',
+    Deleted = 'invoice.deleted'
```

Comment:
Here I'd recommend something like InvoiceCompleted, InvoiceCancelled, InvoiceExpired, InvoiceDeleted, that's because we'll have more webhook events in the future, not related to invoices.